priv/templates/nimble_template/.github/wiki/Getting-Started.md.eex

```diff
@@ -57,6 +57,16 @@
   mix test
   ```
 
+- Run all tests with `watch` option:
```

Comment:
Because there are some options to run the test in a specific file/line of file so it should be
```suggestion
- Run tests with `watch` option:
```
What do you think @byhbt

Reply:
thanks @nvminhtue ! Updated here: https://github.com/nimblehq/elixir-templates/pull/247/commits/5dca0d2b4a4515f5bb38338b303ce2e3a488ad13